tests/test_benchmark.py

```diff
@@ -1,11 +1,11 @@
-import os
-import time
-import timeit
 import unittest
 
-import numpy as np
 from sparselsh import LSH
-from scipy.sparse import csr_matrix, issparse
+from scipy.sparse import csr_matrix
+import numpy as np
+
+import os
+import timeit
```

Comment:
The imports here should follow the previous ordering, here they're the opposite of how they should be. https://peps.python.org/pep-0008/#imports

```
Imports should be grouped in the following order:

    Standard library imports.
    Related third party imports.
    Local application/library specific imports.
```

Reply:
Done. Thanks for the guidelines.